src/sass/_network_detail_list.scss

```diff
@@ -0,0 +1,12 @@
+.networks {
```

Comment:
This is referencing a class that is not in the NetworkListTable component. We can define a dedicated `network-table` class in it and reference it here.

Then move the .networks and border-bottom to the respective usages in profile overview and instance overview scss files. Because the border is referencing element in those components.

The main idea is to keep the scss files aligned with the react component files and avoid referencing classes from other components.